### ISSUE TITLE
TextFieldBlocBuilder in Row

### ISSUE THREAD

**Issue by sm2017** [GiancarloCode/form_bloc#147]
@GiancarloCode I wrap `TextFieldBlocBuilder` in `Expanded` , `Flexible`  and `SizedBox` with width , To render `TextFieldBlocBuilder` in `Row`

If field has no initial value , there is no problem , But when has initial value it get hidden!!

```dart
            Row(
                  mainAxisAlignment: MainAxisAlignment.center,
                  children: <Widget>[
                    SizedBox(
                      width: 100,
                      height: 100,
                      child: TextFieldBlocBuilder(
                        textFieldBloc: formBloc.foo,
                        style: const TextStyle(fontSize: 14),
                        keyboardType: TextInputType.number,
                        textInputAction: TextInputAction.next,
                        onSubmitted: nextFocus,
                      ),
                    ),
                    SizedBox(
                      width: 100,
                      height: 100,
                      child: TextFieldBlocBuilder(
                        textFieldBloc: formBloc.bar,
                        style: const TextStyle(fontSize: 14),
                        keyboardType: TextInputType.number,
                        textInputAction: TextInputAction.next,
                        onSubmitted: nextFocus,
                      ),
                    ),
                  ],
                )
```

**Comment by sm2017** [comment]
@GiancarloCode Reply please

**Comment by sm2017** [comment]
@GiancarloCode Sorry for mention again, Can you help me?

**Comment by sm2017** [comment]
😭

**Comment by GiancarloCode** [comment]
Hello @sm2017 , sorry for the delay, I cannot reproduce the error, can you please make a minimal functional example.
I leave you the example with which I tried.

<details>
<summary>Example</summary>

```dart

import 'package:flutter/material.dart';
import 'package:flutter_form_bloc/flutter_form_bloc.dart';

void main() {
  runApp(App());
}

class App extends StatelessWidget {
  const App({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      debugShowCheckedModeBanner: false,
      home: AllFieldsForm(),
    );
  }
}

class AllFieldsFormBloc extends FormBloc<String, String> {
  final text1 = TextFieldBloc(initialValue: '1');
  final text2 = TextFieldBloc(initialValue: '1');

  AllFieldsFormBloc() {
    addFieldBlocs(fieldBlocs: [
      text1,
      text2,
    ]);

    text1.updateInitialValue('2');
    text2.updateInitialValue('2');
  }

  @override
  void onSubmitting() async {
    try {
      await Future<void>.delayed(Duration(milliseconds: 500));

      emitSuccess(canSubmitAgain: true);
    } catch (e) {
      emitFailure();
    }
  }
}

class AllFieldsForm extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return BlocProvider(
      create: (context) => AllFieldsFormBloc(),
      child: Builder(
        builder: (context) {
          final formBloc = BlocProvider.of<AllFieldsFormBloc>(context);

          return Theme(
            data: Theme.of(context).copyWith(
              inputDecorationTheme: InputDecorationTheme(
                border: OutlineInputBorder(
                  borderRadius: BorderRadius.circular(20),
                ),
              ),
            ),
            child: Scaffold(
              appBar: AppBar(title: Text('Example')),
              body: FormBlocListener<AllFieldsFormBloc, String, String>(
                onSubmitting: (context, state) {
                  LoadingDialog.show(context);
                },
                onSuccess: (context, state) {
                  LoadingDialog.hide(context);

                  Navigator.of(context).pushReplacement(
                      MaterialPageRoute(builder: (_) => SuccessScreen()));
                },
                onFailure: (context, state) {
                  LoadingDialog.hide(context);

                  Scaffold.of(context).showSnackBar(
                      SnackBar(content: Text(state.failureResponse)));
                },
                child: SingleChildScrollView(
                  physics: ClampingScrollPhysics(),
                  child: Padding(
                    padding: const EdgeInsets.all(24.0),
                    child: Column(
                      children: <Widget>[
                        Row(
                          mainAxisAlignment: MainAxisAlignment.center,
                          children: <Widget>[
                            SizedBox(
                              width: 100,
                              height: 100,
                              child: TextFieldBlocBuilder(
                                textFieldBloc: formBloc.text1,
                                style: const TextStyle(fontSize: 14),
                                keyboardType: TextInputType.number,
                                textInputAction: TextInputAction.next,
                              ),
                            ),
                            Expanded(
                              child: TextFieldBlocBuilder(
                                textFieldBloc: formBloc.text1,
                                style: const TextStyle(fontSize: 14),
                                keyboardType: TextInputType.number,
                                textInputAction: TextInputAction.next,
                              ),
                            ),
                          ],
                        ),
                      ],
                    ),
                  ),
                ),
              ),
            ),
          );
        },
      ),
    );
  }
}

class LoadingDialog extends StatelessWidget {
  static void show(BuildContext context, {Key key}) => showDialog<void>(
        context: context,
        useRootNavigator: false,
        barrierDismissible: false,
        builder: (_) => LoadingDialog(key: key),
      ).then((_) => FocusScope.of(context).requestFocus(FocusNode()));

  static void hide(BuildContext context) => Navigator.pop(context);

  LoadingDialog({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return WillPopScope(
      onWillPop: () async => false,
      child: Center(
        child: Card(
          child: Container(
            width: 80,
            height: 80,
            padding: EdgeInsets.all(12.0),
            child: CircularProgressIndicator(),
          ),
        ),
      ),
    );
  }
}

class SuccessScreen extends StatelessWidget {
  SuccessScreen({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: <Widget>[
            Icon(Icons.tag_faces, size: 100),
            SizedBox(height: 10),
            Text(
              'Success',
              style: TextStyle(fontSize: 54, color: Colors.black),
              textAlign: TextAlign.center,
            ),
            SizedBox(height: 10),
            RaisedButton.icon(
              onPressed: () => Navigator.of(context).pushReplacement(
                  MaterialPageRoute(builder: (_) => AllFieldsForm())),
              icon: Icon(Icons.replay),
              label: Text('AGAIN'),
            ),
          ],
        ),
      ),
    );
  }
}

```
</details>

**Comment by sm2017** [comment]
@GiancarloCode Thanks a lot, Unfortunately I'm not able to create a minimal reproducible example!!
May be something wrong in my side or may be it's hard to reproduce it!

Let me try again!
Currently I'm working in another project, In free time I will create and send you a minimal reproducible example
